### PR TITLE
feat(rpc): Updated quorum listextended RPC

### PR DESCRIPTION
**Pull request by ogabrielides**
## Issue being fixed or feature implemented

- At the request of Platform team, this RPC should accept `height` instead of `count`. RPC replies with the signing quorums active at requested `height`. If `height` isn't specified, then tip is used for the construction of the reply.
- Corrections were made on the description of reply model.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by ogabrielides** [comment]
@PastaPastaPasta Technically no because `quorum listextended` was never released :)

**Review by thephez — APPROVED** [review]
:+1:

**Comment by UdjinM6** [comment]
hmm.. can't we have both `count` and `height`?

**Comment by ogabrielides** [comment]
> hmm.. can't we have both `count` and `height`?

1. Platform always want signing count
2. `count` and `height` are both optional integer. It would be confusing no?

**Comment by UdjinM6** [comment]
> 2. `count` and `height` are both optional integer. It would be confusing no?

good point

**Comment by PastaPastaPasta** [comment]
Should also add/edit release notes

**Comment by ogabrielides** [comment]
> Should also add/edit release notes

Updated already existing one

File: src/rpc/quorums.cpp [hunk 1]
```diff
@@ -87,25 +87,27 @@ static void quorum_list_extended_help(const JSONRPCRequest& request)
...
+                        {RPCResult::Type::OBJ, "","",
+                        {
                         {RPCResult::Type::OBJ, "xxxx", "Quorum hash. Note: most recent quorums come first.",
-                         {
+                        {
                                  {RPCResult::Type::NUM, "creationHeight", "Block height where the DKG started."},
                                  {RPCResult::Type::NUM, "quorumIndex", "Quorum index (applicable only to rotated quorums)."},
                                  {RPCResult::Type::STR_HEX, "minedBlockHash", "Blockhash where the commitment was mined."}
-                         }},
+                         }}}},
```
Comment (UdjinM6) [inline src/rpc/quorums.cpp:104]: format:
```suggestion
                    {RPCResult::Type::OBJ, "", "",
                    {
                        {RPCResult::Type::OBJ, "xxxx", "Quorum hash. Note: most recent quorums come first.",
                        {
                            {RPCResult::Type::NUM, "creationHeight", "Block height where the DKG started."},
                            {RPCResult::Type::NUM, "quorumIndex", "Quorum index (applicable only to rotated quorums)."},
                            {RPCResult::Type::STR_HEX, "minedBlockHash", "Blockhash where the commitment was mined."}
                        }}
                    }},
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**